app/Models/FeedDAO.php

```diff
@@ -1,16 +1,17 @@
 <?php
 
-class FreshRSS_FeedDAO extends Minz_ModelPdo {
+class FreshRSS_FeedDAO extends Minz_ModelPdo
+{
 
 	protected function addColumn(string $name): bool {
 		if ($this->pdo->inTransaction()) {
 			$this->pdo->commit();
 		}
 		Minz_Log::warning(__method__ . ': ' . $name);
 		try {
-			if ($name === 'kind') {	//v1.20.0
+			if ($name === 'kind') {    //v1.20.0
```

Comment:
Please keep the whitespace as it was for such minor things (several similar cases below)